### DISCUSSION TITLE
Azure: All Containers in Storage Account has atRestEncryption enabled?

### DISCUSSION THREAD

**Discussion by lebogg** [clouditor/clouditor#883] [Q&A] [answered]
I got some feedback about the UI. Among others, it was that all containers of an Azure Storage Account has encryption enabled (or none). So it stays the same for all containers. Is this true? @anatheka maybe?

An idea would be, e.g., to add a hint in the UI that all storage account containers have at rest encryption enabled or not -  to reduce the space possible tons of redundant information container resources take.

**Comment by anatheka** [comment, marked as answer]
In Azure all containers are encrypted at rest by default with AES256, but it also possible to encrypt blobs and containers with encryption scopes. But that isn't implemented right now. Currently, each container in the storage account has the same atRestEncryption information.
